src/components/UserUpload/BulkUser.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useCallback } from 'react';
```

Comment:
We updated this component on Onboarding (pending merge) but once merged you guys can add the 'if (file.name.substr(file.name.length - 3) === 'csv')' and the updates to render a successful users upload table.